constellations/gemini-user/src/internship/internship.sass

```diff
@@ -0,0 +1,56 @@
+@import "../utils/constants.sass"
+
+#view-intern
+    font-family: $fontFamilyRoboto
+    width: 65%
+    margin-left: auto
+    margin-right: auto
+
+
+    & h1
+        color: $colorTurquoise
+        margin-top: 30px
+        margin-bottom: 30px
```

Comment:
Take a look at this file: https://github.com/ahsu1230/mathnavigatorSite/blob/master/resources/frontend/css_ordering.md
It lays out the conventional ordering for CSS properties. So, in this case, margin-top and margin-bottom would come before color.